platform/darwin/src/MGLCircleStyleLayer.h

```diff
@@ -55,8 +55,8 @@ typedef NS_ENUM(NSUInteger, MGLCircleTranslateAnchor) {
     22: MGLStyleValue(rawValue: 180)
  ])
  layer.circleOpacity = MGLStyleValue(rawValue: 0.7)
- layer.predicate = NSPredicate(format: "%K == %@", "ethnicity", "ewok")
- mapView.style().add(layer)
+ layer.predicate = NSPredicate(format: "%K == %@", "marital-status", "married")
+ mapView.style.addLayer(layer)
```

Comment:
You’re referring to the change from `add(_:)` to `addLayer(_:)`? I was curious about that too. It’s unlikely to be due to the change from `MGLMapView.style()` to `MGLMapView.style`. The more likely cause is that #6097 added `layers` and `sources` properties, or that we started using Xcode 8.2 instead 8.1, but I’m not really sure exactly what introduced the change.

As for whether we should use `NS_SWIFT_NAME`, I’m ambivalent. If we do, we should make sure to refine all the layer and source methods on this class.